test/redis_test.rb

```diff
@@ -19,9 +19,9 @@ def setup
   end
 
   def test_semian_identifier
-    assert_equal :redis_foo, Redis.new(semian: {name: 'foo'}).client.semian_identifier
-    assert_equal :'redis_127.0.0.1:6379/0', Redis.new.client.semian_identifier
-    assert_equal :'redis_example.com:42/0', Redis.new(host: 'example.com', port: 42).client.semian_identifier
```

Comment:
The semian resource used to be lazily accessed, so this test was able to work without providing semian config.

Now that it's eagerly accessed, it's not possible anymore, so I had to fix that test.